test/index.test.js

```diff
@@ -1,13 +1,52 @@
+/*jshint expr: true*/
+
+import { base, example } from 'feathers-service-tests';
+import errors from 'feathers-errors';
+import feathers from 'feathers';
 import assert from 'assert';
-import plugin from '../src';
+import server from './test-app';
+import service from '../src';
+import localstorage from 'localstorage-memory';
+
+const _ids = {};
+const app = feathers().use('/people', service({
+  storage: localstorage
+}));
+const people = app.service('people');
+
+function clean() {
+  people._uId = 0;
+  people.store = {};
+}
+
+describe('Feathers Localstorage Service', () => {
+  before(clean);
+  after(clean);
+
+  beforeEach(done => {
+    people.create({
+      name: 'Doug',
+      age: 32
+    }).then(data => {
+      _ids.Doug = data.id;
+      done();
+    }, done);
+  });
+
+  afterEach(done => {
+    const doneNow = () => done();
+    people.remove(_ids.Doug).then(doneNow, doneNow);
+  });
 
-describe('feathers-localstorage', () => {
   it('is CommonJS compatible', () => {
```

Comment:
We might want to set up `after()` to make sure that something has been written to the storage.